File: eTuristickaAgencija.Models/Rezervacija.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTuristickaAgencija.Models
{

    public class Rezervacija
    {
        public int Id { get; set; }
        public decimal Cijena { get; set; }
        public int? HotelId { get; set; }
        public bool? Otkazana { get; set; }
        public DateTime DatumRezervacije { get; set; }
        public int? KorisnikId { get; set; }

        public static implicit operator List<object>(Rezervacija v)
        {
            throw new NotImplementedException();
        }
    }
}

File: UI/eturistickaagencija_mobile/lib/model/rezervacija.dart
class Rezervacije {
  final int? id;
  int hotelId;
  int korisnikId;
  DateTime datumRezervacije;
  bool otkazana;
  double cijena;

  Rezervacije({
    this.id,
    required this.hotelId,
    required this.korisnikId,
    required this.datumRezervacije,
    required this.otkazana,
    required this.cijena,
  });

  Map<String, dynamic> toJson() {
    return {
      'id':id,
      'hotelId': hotelId,
      'korisnikId': korisnikId,
      'datumRezervacije': datumRezervacije.toIso8601String(),
      'otkazana': otkazana,
      'cijena': cijena,
    };
  }
}
